functions/tokens/[[index]].ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-unused-modules */
+import { MetaTagInjector } from '../components/metaTagInjector'
+import getToken from '../utils/getToken'
+
+const convertTokenAddress = (tokenAddress: string) => {
+  return tokenAddress && tokenAddress === 'NATIVE' ? '0x0000000000000000000000000000000000000000' : tokenAddress
+}
+
+export const onRequest: PagesFunction = async ({ params, request, next }) => {
+  const { index } = params
+  const networkName = index[0]?.toString().toUpperCase()
+  const tokenAddress = convertTokenAddress(index[1]?.toString())
+  if (!tokenAddress) {
```

Comment:
what if `networkName` is not defined?

Reply:
I don't believe this function ever gets called if `networkName` is undefined.